.mergify.yml

```diff
@@ -0,0 +1,77 @@
+pull_request_rules:
```

Comment:
Should this mean that we must create backports for PRs affecting this file, right?

Reply:
IIRC, mergify uses the default branch, so therefore this file is meaningless in terms of being used for the mergify backport automation. 

We can change this PR if needed and delete the file if it helps, though I dont' see any issues if it's backported as it is now, what do you prefer?

Reply:
I'm totally OK with the file being backported. I asked just in case the bump automation needed it for some reason